ci/kokoro/install/build.sh

```diff
@@ -17,15 +17,71 @@
 set -eu
 
 echo "================================================================"
-echo "Change working directory to project root."
+echo "Change working directory to project root $(date)."
 cd "$(dirname "$0")/../../.."
+
+
+echo "================================================================"
+echo "Load Google Container Registry configuration parameters $(date)."
+
+if [[ -f "${KOKORO_GFILE_DIR:-}/gcr-configuration.sh" ]]; then
+  source "${KOKORO_GFILE_DIR:-}/gcr-configuration.sh"
+fi
+
 echo "================================================================"
+echo "Setup Google Container Registry access  $(date)."
+if [[ -f "${KOKORO_GFILE_DIR:-}/gcr-service-account.json" ]]; then
+  gcloud auth activate-service-account --key-file \
+    "${KOKORO_GFILE_DIR}/gcr-service-account.json"
+fi
+gcloud auth configure-docker
 
+readonly DEV_IMAGE="gcr.io/${PROJECT_ID}/google-cloud-cpp/test-install-${DISTRO}"
 echo "================================================================"
-printenv
+echo "Download existing image (if available) for ${DISTRO} $(date)."
+has_cache="/bin/false"
```

Comment:
Why "/bin/"?

Reply:
No good reason, removed here and elsewhere in the script.